tests/modeladmin/tests.py

```diff
@@ -298,6 +298,21 @@ class BandAdmin(ModelAdmin):
             list(list(ma.get_formsets_with_inlines(request))[0][0]().forms[0].fields),
             ['main_band', 'opening_band', 'day', 'id', 'DELETE'])
 
+    def test_overriding_get_exclude(self):
+        """
+        Ensure that overriding ModelAdmin.get_exclude method works
+        Refs #24941.
```

Comment:
I think the ticket reference isn't needed -- the functionality is straightforward and the ticket doesn't really add additional information (we usually reserve ticket references for more obscure issues).